valgrind-xephyr.sh

```diff
@@ -10,6 +10,55 @@ die() {
 
 set -e
 
+cmdname="$0"
+usage() {
+cat <<EOF
+Usage: $cmdname FLAGS
+
+Start a Xephyr and run herbstluftwm in it. Per default, valgrind
+is used. Other debug environments are:
+
+    --valgrind          Run in valgrind (default)
```

Comment:
Hmmm, the default doesn't work as advertised. Is the `|''` case supposed to achieve this? I'm not entirely sure but I wouldn't expect this `for` loop to run at all if there are no arguments.

I think you could move the `debugger() { valgrind "$@" }` in front of the loop and do nothing in the `--valgrind` case. Bash doesn't mind function redefinitions.

Reply:
Oh you're right! Originally I've only checked `$1` and later turned it into a loop. I've fixed this, thanks!

Reply:
Works now! :+1: But I think you forgot to remove the `|''` from the case

Reply:
Gnah, sorry. I've removed this. Thanks!